service/src/main/kotlin/fi/espoo/evaka/reservations/ReservationQueries.kt

```diff
@@ -125,6 +99,14 @@ fun Database.Transaction.clearReservationsForRange(childId: ChildId, range: Date
         .execute()
 }
 
+fun Database.Transaction.deleteAllCitizenReservationsInRange(range: FiniteDateRange) {
+    this.createUpdate(
+            "DELETE FROM attendance_reservation WHERE created_by IN (SELECT id FROM evaka_user WHERE type = 'CITIZEN') AND between_start_and_end(:range, date)"
```

Comment:
This should check that range.start is in the future to avoid wiping out history accidentally

Reply:
I might add that also to frontend as a validation rule

Reply:
Added validation to both frontend and backend

Reply:
Added validation to both frontend and backend